README.md

```diff
@@ -66,7 +66,7 @@ Usage: npm <command> blah blah...
 ```
 
 ## One more thing
-You can notice good news when you didn't know that npm latest version was updated.
+npmbrew uses [update-notifier](https://github.com/yeoman/update-notifier/), to let users know when there is a new version available for download.
```

Comment:
That's correct!
+ Also I used update-notifier both
+ The latter one sould write at here